sdk/communication/azure-communication-email/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 1.0.0b3 (Unreleased)
+## 1.0.0 (2023-03-31)
 
-### Features Added
```

Comment:
Maybe you want to keep it?

Reply:
No new features have been added from the previous beta version, should I just leave these in here but blank?

Reply:
To help me understand better, you listed

The public release of the Azure Communication Services SDK for Email has the following features:
- send emails with a variety of options (multiple recipients, attachments, etc.)
- poll for the status of the email that was sent to track its progress

if they are not new features, does that mean some other features in old betas are removed?

e.g. from b2:
- Adding support for AAD token authentication
- Added the ability to specify the API version by an optional `api_version` keyword parameter.

Reply:
I just reiterated the features that we already have. There are no changes between 1.0.0b2 and 1.0.0.

 I guess I shouldn't add this in? I wasn't totally sure what to put in here.

Reply:
The only thing we need to take into account is if there is no header, it will not be auto-grabbed into our release notes.

I am open to it if you are ok.

Reply:
What header should I add if I would like to be in the release notes?

Reply:
It looks to me ### Features Added is the closest one. :)

(We ignore ### Other Changes)

Reply:
I've updated this to include the Features Added section and have left it blank. Is this what you are looking for here?